api/app/publish/csv.py

```diff
@@ -71,7 +71,7 @@
         "serviceStatus",
         "endpointURL",
         "endpointDescription",
-        "servesData",
+        "servesDataset",
```

Comment:
These are the columns from the spreadsheet which, in the sheet Alasdair linked, are still using the old names - this list is used for validation of the original csv input and your change to `keys_to_rename` takes care of the renaming.
```suggestion
        "servesData",
```